skyportal/utils/offset_util.py

```diff
@@ -1,32 +1,137 @@
+import io
+import os
+from pathlib import Path
 import datetime
+import tempfile
+import hashlib
+import warnings
 
+import pandas as pd
+import requests
+import matplotlib.pyplot as plt
+import seaborn as sns
 import numpy as np
+from scipy.ndimage.filters import gaussian_filter
+
 from astropy import units as u
 from astropy.coordinates import SkyCoord
 from astroquery.gaia import Gaia
 from astropy.time import Time
+from astropy.utils.exceptions import AstropyWarning
+
+from astropy.wcs import WCS
```

Comment:
This style of importing makes it quite hard to figure out where functions are coming from later.  Not a deal-breaker, but consider the long form of the import:
```
import astropy
import astropy.wcs
etc.
```

Reply:
I'm going to leave the importing as is for now. We can do some unification of importing style in another PR (across the project).